mergify_engine/branch_updater.py

```diff
@@ -159,7 +159,10 @@ def _do_update(ctxt, token, method="merge"):
     except subprocess.CalledProcessError as in_exception:  # pragma: no cover
         for message, out_exception in GIT_MESSAGE_TO_EXCEPTION.items():
             if message in in_exception.output:
-                raise out_exception(in_exception.output.decode())
+                raise out_exception(
+                    "Git reported the following error:\n\n"
+                    f"```{in_exception.output.decode()}```"
```

Comment:
```suggestion
                    "Git reported the following error:\n"
                    f"```\n{in_exception.output.decode()}```"
```

Reply:
~I don't get it~ ok I get it now :smile: 

mine: 
<hr>
Git reported the following error:
 
```{in_exception.output.decode()}```
<hr>

yours: 
<hr>

Git reported the following error:
```
{in_exception.output.decode()}
```

<hr>